lib/jsdom/living/nodes/Element-impl.js

```diff
@@ -409,6 +409,35 @@ class ElementImpl extends NodeImpl {
     }
   }
 
+  insertAdjacentElement(position, element) {
+    position = position.toLowerCase();
+
+    if (this.parentNode === null || this.parentNode.nodeType === NODE_TYPE.DOCUMENT_NODE) {
+      throw new DOMException(DOMException.NO_MODIFICATION_ALLOWED_ERR, "Modifications are not allowed for this" +
+      " document");
+    }
+    switch (position) {
```

Comment:
Thanks for this...
If anyone else stumbles across this. Try dropping this switch into your setup file for jest  ( which you declare in package.json ) into the following function:

```
HTMLElement.prototype.insertAdjacentElement = function (position, element) {
  // switch here
  return element
}
```